File: AD-HOC/2567/main.js
function maxLethality(n, a) {
    a = a.sort((p, q) => p - q);
    let lethality = 0;
    for (let i = 0; i < parseInt(n / 2); i++) {
        lethality += a[n - 1 - i] - a[i];
    }
    return lethality;
}

let lines = require('fs').readFileSync('/dev/stdin', 'utf-8').trim().split('\n');
let line;
while (line = lines.shift()) {
    let n = parseInt(line);
    let a = lines.shift().split(' ').map(Number);
    console.log(maxLethality(n, a));
}

File: AD-HOC/2567/main.rb
# frozen_string_literal:true

def max_lethality(n, a)
  a.sort!
  lethality = 0
  (0...n / 2).each do |i|
    lethality += a[n - 1 - i] - a[i]
  end
  lethality
end

while (n = gets&.to_i)
  a = gets.split.map(&:to_i)
  puts max_lethality(n, a)
end
